source/validations.ts

```diff
@@ -31,6 +31,28 @@ class ValidationError extends Error {
 	}
 }
 
+/**
+ * A warning logged when the configuration used will/has been changed by a
+ * newly released version of the library.
+ */
+class ChangeWarning extends ValidationError {
+	name: string
+
+	/**
+	 * The code must be a string, in snake case and all capital, that starts with
+	 * the substring `WRN_ERL_`.
+	 *
+	 * The message must be a string, starting with an uppercase character,
+	 * describing the issue in detail.
+	 */
+	constructor(code: string, message: string) {
+		super(code, message)
+
+		// `this.constructor.name` is the class name
+		this.name = this.constructor.name
```

Comment:
Is this needed? I think the one in ValidationError would set it correctly. (Because `this` will be the ChangeWarning instance even in the ValidationError constructor (?)

in fact, I'm not sure we even need a constructor.

Reply:
It's needed since it sets the name of the error to `ChangeWarning`.

Reply:
If we don't call it here I think the name would be `ValidationError`.

Reply:
Turns out I am wrong, the constructor and this stuff isn't needed. Will change it, thanks!